app.rb

```diff
@@ -24,6 +24,8 @@ class App < Sinatra::Base
   end
 
   post '/user-signup/sms-notification' do
+    logger.info("Processing SMS on /user-signup/sms-notification from #{params[:source]} with message #{params[:message]}")
```

Comment:
It should be pretty clear that it is coming from that URL, but that's a minor point.